ONSdigital/design-system
# Update babel preset env, core and autoprefixer

### What is the context of this PR?

Fixes: ONSdigital/design-team#56

This updates `@babel/preset-env`, `@babel/core` and `autoprefixer` to remove the caniuse-lite warning.

### How to review this PR

- Run `yarn start` on main, see that the warning is printed in the terminal
- Run `yarn start` on this branch, see that the warning is now gone

### Checklist

This needs to be completed by the person raising the PR.
